sentry-logback/build.gradle.kts

```diff
@@ -0,0 +1,93 @@
+import com.novoda.gradle.release.PublishExtension
+
+plugins {
+    `java-library`
+    kotlin("jvm")
+    jacoco
+    id(Config.QualityPlugins.errorProne)
+    id(Config.Deploy.novodaBintray)
+    id(Config.QualityPlugins.gradleVersions)
+}
+
+configure<JavaPluginConvention> {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach {
+    kotlinOptions.jvmTarget = JavaVersion.VERSION_1_8.toString()
+}
+
+dependencies {
+    implementation(project(":sentry-core"))
```

Comment:
@marandaneto I am not 100% sure about scope of these dependencies.

Reply:
the timber integration has it as an `api` instead of `implementation` because one may configure events and breadcrumbs min level, so the client needs access to sentry-core classes, if in this case, they dont need to access any of the Sentry classes that are bundled in sentry-core, then `implementation` is fine

Reply:
Wouldn't we document that users using logback can add only this package and get the whole `Sentry.X` API?
Otherwise users would need to make sure they bump both `sentry-core` and `sentry-logback` while keeping them compatible.

I believe it would make sense to have all the transient package's API show up. So `api` like the `timber` makes sense?